src/Resources/views/Download/creditMemo.html.twig

```diff
@@ -10,7 +10,42 @@
 <h1>{{ 'sylius_refund.ui.credit_memo'|trans }} #{{ creditMemo.number }}</h1>
 <p>{{ 'sylius_refund.ui.order_number'|trans }}: #{{ creditMemo.orderNumber }}</p>
 
+{% if creditMemo.from != null or creditMemo.to != null %}
 <br /><br />
+{% if creditMemo.from != null %}
+<p>
+    {% set from = creditMemo.from %}
+    <h4>{{ 'sylius_refund.ui.from'|trans }}</h4>
+    <address>
+        <strong>{{ from.customerName }}</strong>
+        {% if from.company %}
+            {{ from.company }}
+        {% endif %}
+        <br/>{{ from.street }}<br/>
+        {{ from.city }}<br/>
+        {% if from.provinceName is not empty %}
+            {{ from.provinceName }}<br/>
+        {% endif %}
+        <i class="{{ from.countryCode|lower }} flag"></i>
+        {{ from.countryCode|sylius_country_name|upper }} {{ from.postcode }}
+    </address>
+</p>
+{% endif %}
+{% if creditMemo.to != null and creditMemo.to.taxId != null %}
+<p>
+    {% set to = creditMemo.to %}
+    <h4>{{ 'sylius_refund.ui.to'|trans }}</h4>
+    <address>
+        <strong>{{ to.company }}</strong>
+        {{ to.taxId }}<br/>
+        {{ to.city }}<br/>
+        {{ to.street }}<br/>
+        <i class="{{ to.countryCode|lower }} flag"></i>
+        {{ to.countryCode|sylius_country_name|upper }} {{ to.postcode }}
+    </address>
+</p>
+{% endif %}
+{% endif %}
```

Comment:
For me adding nested if statements without additional tabs look kind of messy. Although I realize that adding them would make the pure html code look not so great, I'd vote for clean code in *.twig files code 🖖 . WDYT?